<launch>
<node pkg="lidar_3d" type="angle_pub.py" name="stepper" launch-prefix="sudo">
</node>

<node pkg="lidar_3d" type="dataPub" name="sensor">
</node>
</launch>



